package.json

```diff
@@ -24,14 +18,19 @@
     "infusion",
     "sonfication"
   ],
-  "repository": "git://github.com/fluid-project/chartAuthoring.git",
+  "repository": "https://github.com/fluid-project/chartAuthoring.git",
   "main": "",
   "engines": {
     "node": ">=0.10.0"
   },
+  "scripts":
+    {
```

Comment:
Fix indentations from line 27-29. This will be captured once the repo is switched to use eslint, which worth a separate JIRA.